docs/current_docs/api/module-structure.mdx

```diff
@@ -9,7 +9,7 @@ import DaggerModuleInit from '../partials/_dagger_module_init.mdx';
 
 <DaggerModuleInit />
 
-## Module layout
+## Physical layout options
```

Comment:
Is physical layout a used term? Here's how nx references it for example. https://nx.dev/concepts/decisions/folder-structure